uint8 action
uint8 action_classification_score